README.md

```diff
@@ -68,7 +82,8 @@ Alternatively, you can manually execute the validations by running `pre-commit r
 
 JCCLI's test suite consists of unit tests and integration tests. The integration tests depend on the environment
 variable `JC_API_KEY`, which should be a Jumpcloud API key corresponding to a blank Jumpcloud instance which can be used
-for testing purposes.
+for testing purposes. The test suite also requires the `HOME` var to be set to the working directory, so that running
```

Comment:
i don't understand this.  my HOME env var is set to `/Users/khai`.  You want me to change my home var to something like `/python/projects/jccli`?  The HOME var is a standard env var in every OS and other things may depend on it as well so I don't want to change that var.

Reply:
Yes, you are correct that I am requiring devs to change the env var `HOME` if they want to run our integration tests. However, they don't have to change it globally.

For example, I can run the tests fine on Ubuntu using a command like the following:
`HOME="/home/connor/PycharmProjects/jccli" JC_API_KEY="123_some_key_789" pytest integration_tests/`
I suppose I should put that in the README, but do you think that's workable for devs?

Reply:
It would be better to make tests depend on some other VAR like `JCCLI_HOME` or something like that.  that way i can set it in my shell and now worry about passing it on each test execution.  

Also what happens if user doesn't set this dependent VAR? will the test just fail to run?

Reply:
> will the test just fail to run?

The way I coded it, yes. If the user doesn't set `HOME` to (some path ending in `/jccli`), then module set-up for the integration tests will fail, and nothing in `integration_tests/` will get run.

My reluctance to use a var like `JCCLI_HOME` is that I would prefer to test actual, normal usage behavior, rather than create some other behavior specifically for the purpose of testing. Unless I'm misunderstanding what you're proposing.

Reply:
well then how about loading a special section in `jccli.ini` file?  something like this..

~/jccli.ini
```
[integration-tests]
key = 1a2b3c4d5e0....
```

Reply:
Yeah, I think that should work fine. Thanks for the suggestion!